server/events-linker.js

```diff
@@ -9,20 +9,22 @@ function catchPromiseWrap (asyncFunction, operationDescription, ...args) {
 }
 
 exports.linkEvents = ({ clockManager, clock, mhub, sounds }) => {
+  const mhubSendPrestartEvent = mhub.sendEvent.bind(null, 'prestart')
   const mhubSendStartEvent = mhub.sendEvent.bind(null, 'start')
   const mhubSendEndEvent = mhub.sendEvent.bind(null, 'end')
   const mhubSendStopEvent = mhub.sendEvent.bind(null, 'stop')
   const mhubSendReloadEvent = mhub.sendEvent.bind(null, 'reload')
 
   clock.on('time', catchPromiseWrap.bind(null, mhub.sendTimeEvent, 'sending time event to mhub'))
 
+  clockManager.on('prestart', catchPromiseWrap.bind(null, mhubSendPrestartEvent, 'sending prestart event to mhub'))
   clockManager.on('start', catchPromiseWrap.bind(null, mhubSendStartEvent, 'sending start event to mhub'))
   clockManager.on('end', catchPromiseWrap.bind(null, mhubSendEndEvent, 'sending end event to mhub'))
   clockManager.on('stop', catchPromiseWrap.bind(null, mhubSendStopEvent, 'sending stop event to mhub'))
   clockManager.on('reload', catchPromiseWrap.bind(null, mhubSendReloadEvent, 'sending reload event to mhub'))
 
   clockManager.on('start', catchPromiseWrap.bind(null, sounds.playStartSound, 'playing start sound'))
   clockManager.on('stop', catchPromiseWrap.bind(null, sounds.playStopSound, 'playing stop sound'))
+  clockManager.on('end', catchPromiseWrap.bind(null, sounds.playEndSound, 'playing end sound'))
```

Comment:
Could you explain why this was changed?

Reply:
This is to make sure the first "0 seconds"event, which occurs when the precount is over, won't play the sound

Reply:
This is to make sure the first "0 seconds"event, which occurs when the precount is over, won't play the sound